modules/client/examples/03-tokenVoting-client/11-get-members.ts

```diff
@@ -18,7 +18,7 @@ const tokenVotingClient: TokenVotingClient = new TokenVotingClient(
   contextPlugin,
 );
 
-const pluginAddress: string = "0x12345384572394756239846529574932532985"; // the address of the plugin that DAO has installed. You can find this through getting the DAO details.
+const pluginAddress: string = "0x12345384572394756239846529574932532985"; //  The address of the plugin that DAO has installed. You can find this by calling `getDao(daoAddress)` and getting the DAO details .
```

Comment:
First, `daoPluginAddress` is a confusing name. Is it the address of the DAO or of the plugin?

Second, the comment is wrong:
> // The address of the plugin that DAO has installed. You can find this by calling `getDao(daoAddress)` and getting the DAO details .

You don't get the `pluginAddress` (which we want to have here) by calling `getDao(daoAddress)`.

Reply:
Actually, getDao() returns an object with a field that contains the list of installed plugins (and their address)